src/main/java/utils/Utility.java

```diff
@@ -2,6 +2,19 @@
 
 public class Utility {
 
+    public static class Temperature{
+
+        public static double Celsius(double fahrenheit){
```

Comment:
methods should be named with first letter as lower case
class - ClassName{}
method - methodName()
variable - variableName;
final variable - FINAL_VARIABLE;

Reply:
public static class Temperature {

        public static double toCelsius(double fahrenheit) {
            return 5.0 / 9.0 * (fahrenheit - 32);
        }

        public static double toFahrenheit(double celsius) {
            return 9.0 / 5.0 * celsius + 32;
        }
    }

fixed